app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,25 @@
 package com.codesoom.assignment;
 
-public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+import com.codesoom.assignment.handler.TaskHttpHandler;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
 
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+public class App {
+    private static final int DEFAULT_PORT = 8000;
+    private static String ROOT_PATH = "/";
     public static void main(String[] args) {
+        try {
+            InetSocketAddress address = new InetSocketAddress(DEFAULT_PORT);
+            HttpServer httpServer = HttpServer.create(address,0);
+            HttpHandler handler = new TaskHttpHandler();
+            httpServer.createContext(ROOT_PATH,handler);
```

Comment:
```suggestion
            httpServer.createContext(ROOT_PATH, handler);
```

콤마 사이에 빈 칸이 없네요. 다른 코드에서는 빈 칸으로 구분하는 것 같아서 일관성있게 맞춰주면 좋을 것 같아요.
에디터의 자동 정렬 기능을 습관하시면 좋습니다. 혹은 파일을 저장할 때 마다 정렬되도록 할 수도 있습니다.

![image](https://github.com/CodeSoom/spring-week1-assignment-1/assets/14071105/e340498c-acc4-48aa-af26-123e7d24b838)